tests/main/desktop-portal-filechooser/task.yaml

```diff
@@ -76,6 +76,6 @@ debug: |
     #shellcheck source=tests/lib/desktop-portal.sh
     . "$TESTSLIB"/desktop-portal.sh
 
-    ls -la "/run/user/$TEST_UID/" || true
+    ls -la "/run/user/$(id -u test)" || true
```

Comment:
Thanks for sending this. I noticed it is broken when I was debugging it recently but haven't sent any fixes yet.